runtime/compiler/codegen/J9CodeGenerator.cpp

```diff
@@ -767,23 +767,27 @@ J9::CodeGenerator::lowerTreeIfNeeded(
 
```

Comment:
Unless I'm mis-reading the logic, I think this code would be more readable (and you would help out the compiler to optimize it better) if you were to write it like this:
```
if (node->getOpCode().isCall() &&
    !node->getSymbol()->castToMethodSymbol()->isHelper())
   {
   TR::RecognizedMethod recognizedMethod = node->getSymbol()->castToMethodSymbol()->getRecognizedMethod();

   if (recognizedMethod == TR::java_lang_invoke_MethodHandle_invokeBasic ||
       recognizedMethod == TR::java_lang_invoke_MethodHandle_linkToStatic ||
       ...
      {
      ...
      }
```
Perhaps even consider a `switch` statement.